packages/suite/src/views/suite/SwitchDevice/DeviceItemLegacy/AddWalletButton.tsx

```diff
@@ -0,0 +1,97 @@
+import styled from 'styled-components';
```

Comment:
This whole folder is just renamed without change (we need compatibility since it's under the flag).